JSON/req-schema.json

```diff
@@ -0,0 +1,140 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/perftool-incubator/multiplex/master/req-schema.json",
+
+    "type": "object",
+    "properties": {
+        "presets": {
+            "type": "object",
+            "patternProperties": {
+                "^[a-zA-Z0-9_-]*$": {
+                    "type": "array",
+                    "minItems": 1,
+                    "uniqueItems": true,
+                    "items": {
+                        "$ref": "#/definitions/mv_param"
+                    }
+                }
+            },
+            "additionalProperties": false
+        },
+        "validations": {
+            "type": "object",
+            "patternProperties": {
+                "^[A-Za-z]+[A-Za-z0-9_-]*$": {
+                    "$ref": "#/definitions/validation_obj"
+                }
+            },
+            "additionalProperties": false
+        },
+        "units": {
+            "type": "object",
+            "patternProperties": {
+                "^[A-Za-z]+[A-Za-z0-9_-]*$": {
+                    "$ref": "#/definitions/unit_obj"
+                }
+            },
+            "additionalProperties": false
+        }
+    },
+    "required": [ "validations" ],
```

Comment:
should there be an `"additionalProperties": false` here?

Reply:
yes, accidentally removed it